File: permission/src/main/java/com/banzhi/permission/SettingServer.java
package com.banzhi.permission;

/**
 * <pre>
 * @author : No.1
 * @time : 2018/5/9.
 * @desciption :
 * @version :
 * </pre>
 */

public interface SettingServer {
    /**
     * 执行
     */
    void execute();

    /**
     * 执行
     *
     * @param requestCode
     */
    void execute(int requestCode);

    /**
     * 取消
     */
    void cancle();
}

File: permission_kt/src/main/java/com/banzhi/permission_kt/SettingServer.kt
package com.banzhi.permission_kt

/**
 *<pre>
 * @author : No.1
 * @time : 2019/8/7.
 * @desciption :
 * @version :
 *</pre>
 */
interface SettingServer {
    /**
     * 执行
     */
     fun execute()


    /**
     * 取消
     */
     fun cancle()


    /**
     * 执行
     *
     * @param requestCode
     */
     fun execute(requestCode: Int)
}
